src/app/shared/user/user.service.ts

```diff
@@ -120,15 +120,11 @@ export class UserService {
     this.getExtendedUserData();
   }
 
-  gravatarUrl(email: string | null, defaultImg: string | null): string | null {
-    if (email) {
-      return this.gravatarService.gravatarUrlForEmail(email, defaultImg);
```

Comment:
I can't remember why we did it here instead of just using the gravatarUrl directly from GitHub/Google. I wonder if there were CORS issues, but that doesn't sound right. Maybe we weren't capturing that info in the DB at the time? At any rate, this seems right.